snuba/settings/settings_ci_distributed.py

```diff
@@ -0,0 +1,9 @@
+import os
```

Comment:
Do you think we can reuse the `settings_test_distributed` file here instead of adding another settings file for ci?